--- Xml.pm	2004-12-22 00:02:18.000000000 +0100
+++ vlc.Xml.pm	2004-12-22 00:31:35.000000000 +0100
@@ -181,7 +181,7 @@
 	$self->{options}{'tagsonly'}=0;
 	$self->{options}{'tags'}='';
 	$self->{options}{'attributes'}='';
-	$self->{options}{'inline'}='';
+	$self->{options}{'inline'}='<emphasis> <userinput> <xref> <ulink> <prompt> <command>';
 	$self->{options}{'doctype'}='';
 
 	$self->{options}{'verbose'}='';
